doc/reference/configuration/configuration_reference.rst

```diff
@@ -634,7 +635,8 @@ iproto.advertise.<peer_or_sharding>.*
 
 .. confval:: iproto_advertise.<peer_or_sharding>.login
 
-    A username that should be used to connect to the current instance.
+    (Optional) A username that should be used to connect to the current instance.
```

Comment:
```suggestion
    (Optional) A username used for connecting to the current instance.
```
Feel free to ignore this edit if it changes the meaning.

Reply:
Changed to `A username used to connect to the current instance.`